Mapify/Utils/Extensions.cs

```diff
@@ -277,6 +277,56 @@ public static void SetBasicMapInfo(this SaveGameData saveGameData, BasicMapInfo
                 saveGameData.SetJObject(SAVE_KEY_NAME, JObject.FromObject(basicMapInfo));
         }
 
+        public static RailTrack GetRailTrack(this RailTrackRegistry registry, string stationID, string yardID, byte trackNumber)
+        {
+            var query = $"[{stationID}]_[{yardID}-{trackNumber:D2}";
+
+            return registry.AllTracks.FirstOrDefault(track => track.name.Contains(query));
+        }
+
+        public static Junction.Branch FindClosestBranch(this Junction junction, Vector3 fromPoint, float maxRange = 5f)
+        {
+            var closestDistance = float.PositiveInfinity;
+
+            RailTrack track = null;
+            var first = false;
+
+            foreach (var foundTrack in Resources.FindObjectsOfTypeAll<RailTrack>())
```

Comment:
This should be `Object.FindObjectsOfType<RailTrack>()`, no?

Reply:
No, I think this was intentional:
> Contrary to Object.FindObjectsOfType this function will also list disabled objects.  

https://docs.unity3d.com/ScriptReference/Resources.FindObjectsOfTypeAll.html

Reply:
This will also find prefabs, including the prefab for vanilla switches, not just tracks that are loaded in the current session. If you need to access prefabs here, please add a comment explaining why. I'm failing to see the reason.